examples/experimental/trial/imagenet_nas_arch_pytorch/const.yaml

```diff
@@ -59,7 +70,7 @@ hyperparameters:
 resources:
   # We recommend 8 slots with V100
   # and at least 16 slots with K80 GPUs.
-  slots_per_trial: 1
+  slots_per_trial: 2
```

Comment:
Question: why two here? It's a const.yaml so i would expect slots_per_trial: 1. If it's due to memory, you can set it to one and put a note in the readme and here the required space.

Reply:
changed back to 1